common/src/main/scala/no/ndla/common/SchemaImplicits.scala

```diff
@@ -0,0 +1,17 @@
+/*
+ * Part of NDLA common
+ * Copyright (C) 2025 NDLA
+ *
+ * See LICENSE
+ *
+ */
+package no.ndla.common
```

Comment:
Veit det er pirk. Kunne vi oppdatert lisensplugin til å passe på dette for oss?
```suggestion

package no.ndla.common
```